sources/packages/api/src/esdc-integration/disbursement-receipt-integration/models/disbursement-receipt-integration.model.ts

```diff
@@ -0,0 +1,19 @@
+import { DisbursementReceiptDetail } from "../disbursement-receipt-files/disbursement-receipt-file-detail";
+import { DisbursementReceiptHeader } from "../disbursement-receipt-files/disbursement-receipt-file-header";
+
+/**
+ * Record type of disbursement receipt records.
+ */
+export enum DisbursementReceiptRecordType {
+  Header = "F",
+  Detail = "D",
+  Trailer = "T",
```

Comment:
The file that handles this record is called `DisbursementReceiptFooter`. Can we call both footers or both trailers? I personally would prefer the footer because I believe that is what we used mostly but both are accurate.